src/components/super/i-static-page/README.md

```diff
@@ -260,8 +260,9 @@ const deps = {
  *   1. `lib` - the external library, i.e, something from `node_modules`
  *   2. `src` - the internal resource, i.e, something that builds from the `/src` folder
  *   3. `output` - the output library, i.e, something that builds to the `/dist/client` folder
+ *   4. `external` - externally hosted library, i.e. from a CDN
  *
- * @typedef {('lib'|'src'|'output')}
+ * @typedef {('lib'|'src'|'output'|'external')}
```

Comment:
I noticed now that it is `@typedef` not `@type`, which means it also needs a name:
```suggestion
 * @typedef {('lib'|'src'|'output'|'external')} LibSource
```